### ISSUE TITLE
What's different between PrismLauncher

### ISSUE THREAD

**Issue by jdev082** [PrismLauncher/PrismLauncher#320]
I remember a few months ago I switched from MultiMC to PolyMC for the built in mod downloader. And here I see PrismLauncher. What additional features differentiate PrismLauncher from MultiMC/PolyMC?

**Comment by 678435021** [comment]
https://prismlauncher.org/wiki/overview/faq/#what-happened

**Comment by dogtopus** [comment]
There was a political disagreement between the rest of the ex-PolyMC (now the PrismLauncher team) team and its founder. This caused the founder to boot pretty much everyone else in the team.

Also like 98% of the new features you saw on the PolyMC side came from this team. PolyMC development progress is quite crippled right now and it will probably take a long time to recover.

Tl;dr PrismLauncher is the active one currently and PolyMC would probably stuck with the current state for a while.

**Comment by anoraktrend** [comment]
Beyond that, this one will likely get a Java installer, (as in, an installer for Java) eventually.

**Comment by cool4funny** [comment]
i mean is polymc and prism launcher between those is prism mc better?

**Comment by jdev082** [comment]
> i mean is polymc and prism launcher between those is prism mc better?

yes.

**Comment by TheKodeToad** [comment]
Yeah, also the Discord server and community in general for PolyMC has pretty much been ruined and is generally quite unpleasant and *phobic. I was banned despite not talking at all. No matter your political views, you probably don't want to hang out with the people there.

In terms of the actual piece of software, they've been making some quite naive changes and are very far behind.

(i know this is old news by now but i thought i should explain it :p)
